### PR TITLE
[ft #159285887] Setup jwt

### PR DESCRIPTION
**Pull request by uchemukolo**
#### What does this PR do?
- JSON web token (JWT)
- Implement token-based authentication 
#### Description of Task to be completed?
- Create token
- setup authentication to verify valid or invalid token
#### How should this be manually tested?
- N/A
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- [#159285887](<https://www.pivotaltracker.com/story/show/159285887>)
#### Screenshots (if appropriate)
- N/A
#### Questions:

### CODE REVIEW

File: server/migrations/database.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ CREATE TABLE IF NOT EXISTS userDetails(
 const entries = `
 DROP TABLE IF EXISTS entries cascade;
```
Comment (fidelisojeah) [inline server/migrations/database.js]: Do not drop the table first.

You could have a different migration meant to just drop your tables.

I imagine you have to add a new table to your schema and you run this and lose all your previous tables (and entries).

File: server/migrations/database.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ const pool = new Pool({
 const userDetails = `
 DROP TABLE IF EXISTS userDetails cascade;
```
Comment (fidelisojeah) [inline server/migrations/database.js]: Don't drop the table first

`CREATE TABLE IF NOT EXISTS` is fine

**Review by fidelisojeah — CHANGES_REQUESTED** [review]

File: server/migrations/database.js (hunk 2 above)
Comment (uchemukolo) [reply server/migrations/database.js]: ok

File: server/migrations/database.js (hunk 1 above)
Comment (uchemukolo) [reply server/migrations/database.js]: oh now i get. I will work on it

Comment (uchemukolo) [reply server/migrations/database.js]: please check the below commit for update

**Review by fidelisojeah — APPROVED** [review]
You can merge this

**Pull request merged**